config/locales/hyrax.en.yml

```diff
@@ -725,7 +726,7 @@ en:
           press_to: Press to
           results_per_page: Number of results to display per page
           show_label: Display all details of
-        works: Your Works
+        works: Works
```

Comment:
Looks like all you need is to add `your_works: Your Works` similar to your_collections you added above at 702.